fulltext/util.py

```diff
@@ -135,7 +135,10 @@ def warn(msg):
 
 def is_windows():
     """True if the platform is Windows."""
-    return 'win' in sys.platform
+    if sys.platform != 'darwin':  # aplle osx is not windows
+        return 'win' in sys.platform
+    else:
+        return False
```

Comment:
I thin this should just be `os.name == 'nt'` (this is what I do in psutil)

Reply:
and what will be next steps?

Reply:
@mushroom2 Can you make the change as Giampaolo suggests. If you then open a new PR I will merge it.

Reply:
ok